shared/protocol/src/test/java/io/pravega/shared/protocol/netty/AppendEncodeDecodeTest.java

```diff
@@ -66,6 +68,7 @@
     private Level origionalLogLevel;
 
     private EventExecutor executor =  new EventExecutor() {
+        private final ScheduledExecutorService schedular  = Executors.newSingleThreadScheduledExecutor();
```

Comment:
There is a typo in the variable name.

Reply:
yes.. its changed now.. i will post the update